stack.go

```diff
@@ -0,0 +1,28 @@
+package errors
+
+import (
+	"runtime"
+
+	"github.com/pkg/errors"
+)
+
+// StackTrace is stack of Frames from innermost (newest) to outermost (oldest).
+type StackTrace struct {
+	errors.StackTrace
+}
+
+// NewStackTrace returns a new StackTrace, it skip the number the given number
```

Comment:
"it skip the number the given number of frame, to avoid include the caller" => "it skips the given number of frames, to avoid including the caller"